### PR TITLE
Support for openSUSE Leap 42.2 docker images

### PR DESCRIPTION
**Pull request by rjfd**
This PR adds opensuse distro support for Ceph jewel version.

I could not follow the available instructions for testing the images, so I tested most daemons manually.
Also, where can I see travis build reports of the PRs?

Thanks,
Signed-off-by: Ricardo Dias <rdias@suse.com>

### CODE REVIEW

File: ceph-releases/jewel/opensuse/Leap_42.2/base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# CEPH VERSION DETAIL: 10.2.4
+
+FROM opensuse:42.2
+MAINTAINER Ricardo Dias "rdias@suse.com"
+
+# Install Ceph
+RUN zypper --no-gpg-checks -n ref
+RUN zypper -n --no-gpg-checks install net-tools
+RUN zypper -n --no-gpg-checks install ceph=10.2.4+git.1481215985.12b091b-4.1 ceph-radosgw=10.2.4+git.1481215985.12b091b-4.1
+RUN zypper -n --no-gpg-checks install rbd-mirror=10.2.4+git.1481215985.12b091b-4.1
```
Comment (ddiss) [inline ceph-releases/jewel/opensuse/Leap_42.2/base/Dockerfile]: Why are you specifying --no-gpg-checks here - I'm a docker novice, but shouldn't "FROM opensuse:42.2" pull in the openSUSE repository key to allow for verification here?

Comment (smithfarm) [inline ceph-releases/jewel/opensuse/Leap_42.2/base/Dockerfile]: I think it triggers an interactive question to the user, whether they want to accept the key.

Comment (rjfd) [inline ceph-releases/jewel/opensuse/Leap_42.2/base/Dockerfile]: @ddiss you are right, the image already has the repository keys from opensuse:42.2

**Comment by rjfd** [comment]
Pushed new changes addressing @ddiss comments

File: ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh [hunk 2]
```diff
@@ -556,20 +562,26 @@ function osd_activate {
     timeout 10  bash -c "while [ ! -e ${OSD_DEVICE} ]; do sleep 1; done"
     chown ceph. ${OSD_JOURNAL}
     if [[ ${OSD_DMCRYPT} -eq 1 ]]; then
-      ceph-disk -v --setuser ceph --setgroup --dmcrypt disk activate $(dev_part ${OSD_DEVICE} 1)
+      ceph-disk -v --setuser ceph --setgroup --dmcrypt disk activate $(dev_part ${OSD_DEVICE} 1) ${CEPH_DISK_ACTIVATE_OPTS}
```
Comment (ddiss) [inline ceph-releases/jewel/ubuntu/14.04/daemon/entrypoint.sh]: shouldn't that be "--setgroup disk  --dmcrypt"? Might as well fix it while your here

**Comment by ddiss** [comment]
Otherwise I think this patchset looks fine.

**Comment by rjfd** [comment]
Pushed a new change to address @ddiss last comment.
Thanks @ddiss for your review.

**Comment by leseb** [comment]
jenkins test this please

**Comment by rjfd** [comment]
@leseb if there is work that needs to be done in the testing infrastructure to successfully test this PR, I can gladly help with that.

**Comment by leseb** [comment]
jenkins test this please

**Comment by leseb** [comment]
@rjfd I think the issue is in our CI currently. @andrewschoen what is this docker-fixes branch that we are trying to checkout? The branch doesn't exist hence the failure.

**Comment by rjfd** [comment]
jenkins test this please

**Comment by andrewschoen** [comment]
jenkins test ceph_ansible2.1-jewel-centos7_cluster

**Comment by andrewschoen** [comment]
@leseb it was the branch of ceph-ansible that allowed for custom registry usage, which has since been merged to ``master`` and ``stable-2.1`` of ceph-ansible. A rebase should fix it up the problem, but it seems that's already been done. :)

**Pull request merged**